metricbeat/module/apache/status/_meta/data.json

```diff
@@ -15,17 +15,17 @@
             "cpu": {
                 "children_system": 0,
                 "children_user": 0,
-                "load": 0.0890841,
-                "system": 0.77,
-                "user": 0.65
+                "load": 0.139381,
+                "system": 1.64,
+                "user": 2.14
             },
             "hostname": "apache",
             "load": {
-                "1": 1.53,
-                "15": 0.53,
-                "5": 0.92
+                "1": 0.31,
+                "15": 0.19,
+                "5": 0.2
             },
-            "requests_per_sec": 0.0119197,
+            "requests_per_sec": 0.00331858,
```

Comment:
This will be rounded to 3 digits in elasticsearch which I think is fine. @tsg WDYT?